src/components/molecules/ChatMessageBox/ChatMessageBox.scss

```diff
@@ -11,27 +13,50 @@
     align-items: center;
   }
 
+  &__inputs {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    align-content: center;
+    flex-grow: 1;
+    margin-right: $spacing--lg;
+
+    .InputField {
+      flex-grow: 1;
+
+      input {
+        // NOTE: this removes screen artefacts of the embedded <input> inside InputField being as wide as the parent
+        margin: 1px;
+      }
```

Comment:
This could be revisited on the `InputField` component itself as artefacts due to internal `input` element are visible elsewhere.